scripts/msys2-geany-release.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import shutil
+from os.path import exists, isfile, join
+from subprocess import check_call
+import re
+
+"""
+This script prepares a Geany release on Windows.
+The following steps will be executed:
+- strip binary files (geany.exe, plugin .dlls)
+- create installer
+"""
+
+if 'GITHUB_WORKSPACE' in os.environ:
+    SOURCE_DIR = os.environ['GITHUB_WORKSPACE']
+    BASE_DIR = join(SOURCE_DIR, 'geany_build')
+else:
+    # adjust paths to your needs ($HOME is used because expanduser() returns the Windows home directory)
+    BASE_DIR = join(os.environ['HOME'], 'geany_build')
+    SOURCE_DIR = join(os.environ['HOME'], 'git', 'geany')
+VERSION="2.0"
+with open(join(SOURCE_DIR, 'configure.ac'), 'r') as f:
+    ver = next((l.split(',')[1].strip(' []') for l in f if l.startswith('AC_INIT')), None)
+    if ver is None:
+        print(f"!! FAILED TO GET VERSION FROM {line}")
```

Comment:
Sorry, m'y initial suggestion had an issue, and you probably didn't see the edit:
```suggestion
        print(f"!! FAILED TO GET VERSION FROM {f.name}")
```